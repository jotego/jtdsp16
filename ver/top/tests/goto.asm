r0=1
r2=2
r3=3
goto label
r0=4
r2=5
r3=6
label:
r1=0xcafe
end:
goto end
