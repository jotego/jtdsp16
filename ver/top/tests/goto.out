r0=0x0001
r1=0xcafe
r2=0x0002
r3=0x0003
rb=0x0000
re=0x0000
j =0x0000
k =0x0000
pr=0x0000
pt=0x0000
i =0x0000
a0=0x000000000
a1=0x000000000
x=0x0000
y=0x00000000
p=0x00000000
c0=0x0000
c1=0x0000
c2=0x0000
auc=0x0000
psw=0x0000
